### DISCUSSION TITLE
how to jump back

### DISCUSSION THREAD

**Discussion by engdorm** [ahrm/sioyek#284] [Q&A] [answered]
If I jumped from one point to another
how do I jump back 
and how can I set it like vim (ctrl-o)

**Comment by ahrm** [comment, marked as answer]
You can press backspace, you can also use `<ctrl-left arrow>` and `<ctrl-right arrow>` to jump forward and backward in history.

> how can I set it like vim (ctrl-o)

Add this to your `keys_user.config`:
```
prev_state <C-o>
```

**Reply by zcyzhou** [reply]
Hi, I'm using the Sioyek 2.0.0 on MacOS. It seems the <backspace> does not work.

**Reply by ahrm** [reply]
I think backspace sends `delete` command on macos, you should still be able to use cmd-left/right to move in history (or bind `prev_state` and `next_state` commands to other keys).
